File: CPP/ElektronikDevice.cpp
#pragma once
#include <iostream>

using namespace std;

class ElektronikDevice {
    private:
        int konsumsiDayaWatt;

    public:
        ElektronikDevice() {

        }

        ElektronikDevice(int konsumsiDayaWatt) {
            this->konsumsiDayaWatt = konsumsiDayaWatt;
        }

        void setKonsumsiDayaWatt(int konsumsiDataWatt) {
            this->konsumsiDayaWatt = konsumsiDataWatt;
        }

        int getKonsumsiDayaWatt() const {
            return konsumsiDayaWatt;
        }

        ~ElektronikDevice() {

        }
};

File: Java/ElektronikDevice.java
public class ElektronikDevice {
    private int konsumsiDayaWatt;

    public ElektronikDevice(){
        this.konsumsiDayaWatt = 0;
    }

    public ElektronikDevice(int konsumsiDayaWatt) {
        this.konsumsiDayaWatt = konsumsiDayaWatt;
    }

    public void setKonsumsiDayaWatt(int konsumsiDayaWatt) {
        this.konsumsiDayaWatt = konsumsiDayaWatt;
    }

    public int getKonsumsiDayaWatt() {
        return konsumsiDayaWatt;
    }
}